### ISSUE TITLE
A11y statement's ratings do not match

### ISSUE THREAD

**Issue by marcusosterberg** [Webperf-se/webperf_core#926]
### URL

https://www.eskilstuna.se

### Webperf_core version

2025.2.0

### What environment are you running?

- [ ] Using GitHub Actions
- [ ] Using Docker Image from Dock Hub
- [x] Local machine
- [ ] Other, please describe your environment in How to reproduce.

### Has anyone else reported it already?

- [x] No, there are no other issues related to this in [Issues](https://github.com/Webperf-se/webperf_core/issues).

### Expected Behavior

Desired behaviour is that when executing `python default.py -t 26 -u https://www.eskilstuna.se -L en -r` the overall rating should be the same as the a11y rating. Since the only thing test 26 evaluates is the accessibility

### Rating: 
- Overall: 4.5
- A11y: 4.0

### How to reproduce

```shell
python default.py -t 26 -L en -u https://www.eskilstuna.se -r
```

### Log output

```shell
# Testing website https://www.eskilstuna.se

## Test: 26 - Accessibility Statement

Started: 2025-02-03 13:57:24
Finished: 2025-02-03 13:58:02

### Rating: 
- Overall: 4.5
- A11y: 4.0

### Review:
 
#### Overall:
- Accessibility statement: https://www.eskilstuna.se/ovrigt/om-webbplatsen/tillganglighetsredogorelser
- Has compliance with the legal requirements with the wording &quot;partially compliant&quot; ( 3.00 rating )
- Correct link to DIGG's notification function ( 5.00 rating )
- Link to the accessibility statement was found on the startpage ( 5.00 rating )
- Looks like it is specifying an audit method ( 5.00 rating )
- Accessibility statement seems to be older than 1 year ( 4.50 rating )
#### A11y:
- Self-indicates that the website is only partially compliant with the legal requirements ( 3.00 rating )
- No unreasonably burdensome claims (12 §) ( 5.00 rating )
```

**Comment by 7h3Rabbit** [comment]
@marcusosterberg We could change to this behavior **BUT** it is not a bug.

The reasoning when creating this test was that there are 2 different things we rate here.
1) The law enforcement of **HAVING** a accessibility statement that is **VALID**.
2) Is the accessibility statement (if found) indicating any accessibility problem(s).

I would personally want to keep the 2 different ratings but I'm open for a discussion :)

**Comment by 7h3Rabbit** [comment]
Please note that it could be argued that following should give a rating of 5 for overall.

> #### Overall:
> - Has compliance with the legal requirements with the wording &quot;partially compliant&quot; ( 3.00 rating )

**Comment by marcusosterberg** [comment]
> Please note that it could be argued that following should give a rating of 5 for overall.
> 
> > #### Overall:
> > 
> > * Has compliance with the legal requirements with the wording "partially compliant" ( 3.00 rating )

Well, yes. One can argue that but since all affected websites had 5 years to fix their websites we could also be opinionated enough to keep it at 3 / 5, or perhaps even less.

I have more issues with #927 not having an a11y sub-rating at all when statement is entirely missing. But that might be that I'm used to be able to summarise all a11y sub-rating for Webperf.se and presenting the average as the a11y score.

Could the default `rating.set_a11y` be 1 and changed upwards if better?

**Comment by cockroacher** [comment]
@marcusosterberg 
Can we close this issue now based on the changes made in the referenced issue?

**Comment by 7h3Rabbit** [comment]
@marcusosterberg OK if I close this as won't do?

**Comment by 7h3Rabbit** [comment]
Closing issue as no response after 3 weeks.
